### PR TITLE
doc: concurrency of gc delete-range

### PR DESCRIPTION
**Pull request by ekexium**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Modify the scope of variable `tidb_gc_concurrency`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [ ] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/tidb/pull/54571

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -2574,9 +2574,13 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 - 默认值：`-1`
-- 范围：`[1, 256]`
+- 范围：`{-1} ∪ [1, 256]`
 - 单位：线程
-- 这个变量用于指定 GC 在[Resolve Locks（清理锁）](/garbage-collection-overview.md#resolve-locks清理锁)步骤中线程的数量。默认值 `-1` 表示由 TiDB 自主判断运行 GC 要使用的线程的数量。
+- 功能：此变量用于控制 GC（垃圾回收）过程中的并发线程数。
+- 影响的 GC 步骤：
+  - [Resolve Locks（清理锁）](/garbage-collection-overview.md#resolve-locks清理锁)：直接使用设定的线程数。
+  - [Delete Range（删除区间）](/garbage-collection-overview.md#Delete-Ranges删除区间)：使用设定值的 1/4 作为线程数。
+- 默认值说明：当设置为 `-1` 时，TiDB 将自动决定适当的线程数。
```
Comment (qiancai) [inline system-variables.md:2583]:
```suggestion
- 该变量用于控制 [GC（垃圾回收）](/garbage-collection-overview.md) 过程中 [Resolve Locks（清理锁）] 和 [Delete Range（删除区间）](/garbage-collection-overview.md#Delete-Ranges删除区间) 的并发线程数。
- 当设置为 `-1` 时，TiDB 将根据负载情况自动决定适当的线程数。
- 当设置为 `[1, 256]` 之间的数时：
    - Resolve Locks（清理锁）直接使用该变量设定值作为线程数。
    - Delete Range（删除区间）使用该变量设定值的 1/4 作为线程数。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: system-variables.md [hunk 2]
```diff
@@ -2574,9 +2574,13 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
...
 - 默认值：`-1`
-- 范围：`[1, 256]`
+- 范围：`{-1} ∪ [1, 256]`
 - 单位：线程
-- 这个变量用于指定 GC 在[Resolve Locks（清理锁）](/garbage-collection-overview.md#resolve-locks清理锁)步骤中线程的数量。默认值 `-1` 表示由 TiDB 自主判断运行 GC 要使用的线程的数量。
+- 该变量用于控制 [GC（垃圾回收）](/garbage-collection-overview.md) 过程中 [Resolve Locks（清理锁）] 和 [Delete Range（删除区间）](/garbage-collection-overview.md#Delete-Ranges删除区间) 的并发线程数。
+- 默认情况下，该变量值为 `-1`，TiDB 将根据负载情况自动决定适当的线程数。
+- 当设置为 `[1, 256]` 之间的数时：
+    - Resolve Locks（清理锁）直接使用该变量设定值作为线程数。
+    - Delete Range（删除区间）使用该变量设定值的 1/4 作为线程数。
```
Comment (qiancai) [inline system-variables.md:2583]:
```suggestion
    - Resolve Locks（清理锁）直接使用该变量值作为线程数。
    - Delete Range（删除区间）使用该变量值的 1/4 作为线程数。
```

File: system-variables.md [hunk 3]
```diff
@@ -2574,9 +2574,13 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 默认值：`-1`
-- 范围：`[1, 256]`
+- 范围：`{-1} ∪ [1, 256]`
 - 单位：线程
-- 这个变量用于指定 GC 在[Resolve Locks（清理锁）](/garbage-collection-overview.md#resolve-locks清理锁)步骤中线程的数量。默认值 `-1` 表示由 TiDB 自主判断运行 GC 要使用的线程的数量。
+- 该变量用于控制 [GC（垃圾回收）](/garbage-collection-overview.md) 过程中 [Resolve Locks（清理锁）] 和 [Delete Range（删除区间）](/garbage-collection-overview.md#Delete-Ranges删除区间) 的并发线程数。
```
Comment (Oreoxmt) [inline system-variables.md:2579]:
```suggestion
- 该变量用于控制[垃圾回收 (GC)](/garbage-collection-overview.md) 过程中 [Resolve Locks（清理锁）](/garbage-collection-overview.md#resolve-locks清理锁)和 [Delete Range（删除区间）](/garbage-collection-overview.md#Delete-Ranges删除区间)的并发线程数。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: system-variables.md [hunk 4]
```diff
@@ -2574,9 +2574,13 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 默认值：`-1`
-- 范围：`[1, 256]`
+- 范围：`{-1} ∪ [1, 256]`
```
Comment (Oreoxmt) [inline system-variables.md:2577]:
```suggestion
- 范围：`-1` 或 `[1, 256]`
```

File: system-variables.md [hunk 5]
```diff
@@ -2574,9 +2574,13 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 默认值：`-1`
-- 范围：`[1, 256]`
+- 范围：`-1` 或 `[1, 256]`
 - 单位：线程
-- 这个变量用于指定 GC 在[Resolve Locks（清理锁）](/garbage-collection-overview.md#resolve-locks清理锁)步骤中线程的数量。默认值 `-1` 表示由 TiDB 自主判断运行 GC 要使用的线程的数量。
+- 该变量用于控制[垃圾回收 (GC)](/garbage-collection-overview.md) 过程中 [Resolve Locks（清理锁）](/garbage-collection-overview.md#resolve-locks清理锁)和 [Delete Range（删除区间）](/garbage-collection-overview.md#delete-ranges删除区间)的并发线程数。
+- 默认情况下，该变量值为 `-1`，TiDB 将根据负载情况自动决定适当的线程数。
```
Comment (cfzjywxk) [inline system-variables.md:2580]: 需补充：从 v8.3.0 开始

File: system-variables.md [hunk 6]
```diff
@@ -2574,9 +2574,14 @@ v5.0 后，用户仍可以单独修改以上系统变量（会有废弃警告）
 - 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
 - 类型：整数型
 - 默认值：`-1`
-- 范围：`[1, 256]`
+- 范围：`-1` 或 `[1, 256]`
 - 单位：线程
-- 这个变量用于指定 GC 在[Resolve Locks（清理锁）](/garbage-collection-overview.md#resolve-locks清理锁)步骤中线程的数量。默认值 `-1` 表示由 TiDB 自主判断运行 GC 要使用的线程的数量。
+- 该变量用于控制[垃圾回收 (GC)](/garbage-collection-overview.md) 过程中 [Resolve Locks（清理锁）](/garbage-collection-overview.md#resolve-locks清理锁)的并发线程数。
+- 从 v8.3.0 开始，该变量也用于控制 [Delete Range（删除区间）](/garbage-collection-overview.md#delete-ranges删除区间)的并发线程数。
```
Comment (qiancai) [inline system-variables.md:2580]:
```suggestion
- 从 v8.3.0 开始，该变量也用于控制 GC 过程中 [Delete Range（删除区间）](/garbage-collection-overview.md#delete-ranges删除区间)的并发线程数。
```

**Review by cfzjywxk — APPROVED** [review]

**Pull request merged**